generate-changelog.py

```diff
@@ -0,0 +1,57 @@
+import requests
+import re
+import datetime
+
+def get_last_release():
+    with open("CHANGELOG.md") as file:
+        data = file.read()
+        m = re.search(r"## \[.*\]", data)
+        return ("".join(m.group(0).rsplit(" ", 1)[1:]))[1:-1]
+
+def get_new_release(old_version):
+    while True:
+        new_release = input("Last release version was {}, \nEnter the new version number: ".format(old_version))
+        if new_release == old_version:
+            print("New release cannot be same as the last release")
+            continue
+        return new_release
+
+def get_changes():
+    changes = []
+    link = "https://api.github.com/repos/sendgrid/open-source-library-data-collector/"
+    releases = requests.get(link + "releases").json()
+    pulls = requests.get(link + "pulls?state=closed&per_page=100").json()
```

Comment:
> You often want to send some sort of data in the URL’s query string. If you were constructing the URL by hand, this data would be given as key/value pairs in the URL after a question mark, e.g. httpbin.org/get?key=val. Requests allows you to provide these arguments as a dictionary of strings, using the params keyword argument.

* http://docs.python-requests.org/en/master/user/quickstart/#passing-parameters-in-urls

```python
params = {
    'state': 'closed',
    'per_page': 100
}
pulls = requests.get(link + "pulls", params=params).json()
```

Although it seems a bit inconsistent to not use `str.format` as you do in many other places in this PR.

```python
pulls = requests.get('{}pulls'.format(link), params=params).json()
```

One slight problem with the line above could be readability by not having a slash to break them up. This could be easily fixed by removing it from the end of `link` and modifying the two `get` requests.

```python
link = "https://api.github.com/repos/sendgrid/open-source-library-data-collector"
releases = requests.get('{}/releases'.format(link)).json()

params = {
    'state': 'closed',
    'per_page': 100
}
pulls = requests.get('{}/pulls'.format(link), params=params).json()

```